### ISSUE TITLE
False negatives (jwtdecode, xzing) that do get found with other tool

### ISSUE THREAD

**Issue by carstenhag** [dipien/bye-bye-jetifier#27]
**Describe the bug**
The tool does not find two libs that still use the legacy support libs.

**To Reproduce**
1. Eliminate legacy android support usages as per bye-bye-jetifier
2. Tool says it's ok and jetifier can be disabled
3. Builds fail (duplicate classes, android.support and androidx class paths being there)
4. Check with other tool https://github.com/plnice/can-i-drop-jetifier and that tool finds the two remaining legacy libraries

**Expected behavior**
Both libraries should be found:
* `    implementation 'me.dm7.barcodescanner:zxing:1.9.13'`
* `    implementation "com.auth0.android:jwtdecode:1.4.0"`  

Both are direct dependencies of the single :app module of the project.

**Screenshots**
See additional context

**Additional context**

:canISayByeByeJetifier output:
```
> Task :canISayByeByeJetifier

=========================================
Project: dcs-app-development-android
=========================================
 * No legacy android support usages found

=========================================
Project: app
=========================================
 * No legacy android support usages found

=========================================
Project: appcenter
=========================================
 * No legacy android support usages found

=========================================
Project: libaddressinput
=========================================
 * No legacy android support usages found

=========================================
Project: picasso-2.71828-jetified
=========================================
 * No legacy android support usages found

=====================================================================================
* No dependencies with legacy android support usages! You can say Bye Bye Jetifier. *
=====================================================================================
```

:canIDropJetifier output:
```
> Task :canIDropJetifier
========================================
Project app
========================================

Cannot drop Jetifier due to following module dependencies:

* app (module)

Cannot drop Jetifier due to following external dependencies:

* com.auth0.android:jwtdecode:1.4.0

* me.dm7.barcodescanner:zxing:1.9.13
```

**Comment by carstenhag** [comment]
Ironically, `jetifier-standalone` says for zxing:

```
 ~/Downloads/jetifier-standalone/bin/jetifier-standalone -i ~/Downloads/zxing-1.9.13.aar -o zxing-1.9.13-jetified.aar
WARNING: [Main] No references were rewritten. You don't need to run Jetifier.
```

So that one might be a false positive, but the jwt one should still be a false negative, as there the androidx migration only happened on the 2.0.0 release: https://github.com/auth0/JWTDecode.Android/releases

**Comment by carstenhag** [comment]
Ah I see you are already working on this, I wanted to expand on this as I analyzed it a bit further.

`me.dm7.barcodescanner:zxing:1.9.13` requires `me.dm7.barcodescanner:core:1.9.13`, so that one does need to be transformed.

```
\--- me.dm7.barcodescanner:zxing:1.9.13       <-- OK, AndroidX ready
     +--- me.dm7.barcodescanner:core:1.9.13  <-- Not OK, legacy android stuff
.....
```

**Comment by maxirosson** [comment]
Hi.  Thanks for reporting this issue. We are working on a fix. The problem is when analyzing transitive dependencies. We expect to release a v1.1.1 with the fix soon.

**Comment by maxirosson** [comment]
v1.1.1 released with this fix.

**Comment by maxirosson** [comment]
Released v1.1.2 with the fix
